README.md

```diff
@@ -5,17 +5,19 @@
 G2
 ==========
 
-G2 is a server, worker and client implementation of [Gearman](http://gearman.org/) in [Go Programming Language](http://golang.org). It contains two sub-packages:
+G2 is a server, worker and client implementation of [Gearman](http://gearman.org/) in [Go Programming Language](http://golang.org).
 
-The client package is used for sending jobs to the Gearman job server,
-and getting responses from the server.
+The client package is used for sending jobs to the Gearman job server and getting responses from the server.
 
 	"github.com/appscode/g2/client"
 
-The worker package will help developers in developing Gearman worker
-service easily.
+The worker package will help developers in developing Gearman worker service easily.
 
 	"github.com/appscode/g2/worker"
+	    
+The gearadmin package is used to implement a Gearman admin client.
```

Comment:
Change it to:

The `gearadmin` package implements a client for Gearman admin protocol.